src/Fabulous/Program.fs

```diff
@@ -58,23 +72,44 @@ module Program =
           Logger = ProgramDefaults.defaultLogger()
           ExceptionHandler = ProgramDefaults.defaultExceptionHandler }
 
+    //TODO when would I use this one? How does it compare to the other stateful* builders?
+    //TODO what's expected for 'arg?
     /// Create a program using an MVU loop
     let stateful (init: 'arg -> 'model) (update: 'msg -> 'model -> 'model) =
         define (fun arg -> init arg, Cmd.none) (fun msg model -> update msg model, Cmd.none)
 
+    //TODO when would I use this one? How does it compare to the other stateful* builders?
+    //TODO please explain the concept Cmd<'msg>
+    //TODO what's expected for 'arg?
     /// Create a program using an MVU loop
     let statefulWithCmd (init: 'arg -> 'model * Cmd<'msg>) (update: 'msg -> 'model -> 'model * Cmd<'msg>) = define init update
 
-    /// Create a program using an MVU loop. Add support for CmdMsg
+    //TODO when would I use this one? How does it compare to the other stateful* builders?
+    //TODO please explain the concept CmdMsg vs. Cmd<'msg>
+    //TODO what's expected for 'arg?
+    /// <summary>
+    /// Create a program using an MVU loop supporting CmdMsg.
+    /// See also
+    /// <seealso href="https://elmprogramming.com/elm-architecture-conclusion.html" />
+    /// ?
+    /// </summary>
     let statefulWithCmdMsg (init: 'arg -> 'model * 'cmdMsg list) (update: 'msg -> 'model -> 'model * 'cmdMsg list) (mapCmd: 'cmdMsg -> Cmd<'msg>) =
         let mapCmds cmdMsgs = cmdMsgs |> List.map mapCmd |> Cmd.batch
         define (fun arg -> let m, c = init arg in m, mapCmds c) (fun msg model -> let m, c = update msg model in m, mapCmds c)
 
+    (*TODO Subscriptions will be started or stopped automatically to match.
+        - I don't understand what that means. What (other?) Subscriptions - or - to match what?*)
```

Comment:
I think we are missing a word here. I guess it should be "... to match the lifecycle of the program".
Subscriptions are now disposed when the program exits, for example when a component is removed from the UI tree.

Reply:
I've updated the doco changing the wording.